.scripts/copyright_notice_check.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
`check_copyright_notice()` would set `exit_status=1` for this file... ;-)

Reply:
Updated in fdf95f0a6c9c2da61a6e9a16ddf3124409ddb7e5.

Reply:
fixed in 0e2b98aa